### PR TITLE
Bugfix version of ciceroscm fixing problem with ocean heat content an…

### PR DESCRIPTION
**Pull request by maritsandstad**
…d ocean heat uptake

- [ ] Tests added
- [ ] Documentation added
- [ ] Example added (in the documentation, to an existing notebook, or in a new notebook)
- [ ] Description in ``CHANGELOG.rst`` added (single line such as: ``(`#XX <https://github.com/openscm/openscm-runner/pull/XX>`_) Added feature which does something``)

### CODE REVIEW

**Review by znicholls — COMMENTED** [review]
lgtm, changelog and then we're done (unless you want to add additional tests)

**Comment by znicholls** [comment]
> However, I guess it opens up for a couple of other questions for instance to do with updating the RCMIP database data

We can't update the RCMIP output now that the paper is accepted. Might have to be one for future papers?

**Comment by maritsandstad** [comment]
Updated the changelog now. No real reason why this took me forever...

File: CHANGELOG.rst [hunk 1]
```diff
@@ -14,6 +14,15 @@ The changes listed in this file are categorised as follows:
     - Fixed: any bug fixes
     - Security: in case of vulnerabilities.
 
+2021-12-16
+-------------------
+
+Fixed
+~~~~~~~
+
+- (`#52 <https://github.com/openscm/openscm-runner/pull/57>` _) Updating CICERO-SCM fortran binary which confused OHC and OHC down to 700 meters in output. Also adding missing components to radiative imbalance. Updating tests to much new version.
```
Comment (znicholls) [inline CHANGELOG.rst:23]:
```suggestion
- (`#52 <https://github.com/openscm/openscm-runner/pull/57>` _) Updating CICERO-SCM fortran binary which confused OHC and OHC down to 700 meters in output. Also adding missing components to radiative imbalance. Updating tests to match new version.
```

**Review by znicholls — APPROVED** [review]
lgtm. Fix conflicts etc. then we're done

File: CHANGELOG.rst [hunk 2]
```diff
@@ -14,6 +14,15 @@ The changes listed in this file are categorised as follows:
     - Fixed: any bug fixes
     - Security: in case of vulnerabilities.
 
+2021-12-16
+-------------------
```
Comment (znicholls) [inline CHANGELOG.rst:18]:
```suggestion
----------
```

File: CHANGELOG.rst [hunk 3]
```diff
@@ -23,6 +33,7 @@ Changed
 - (`#58 <https://github.com/openscm/openscm-runner/pull/58>`_) Update README for FaIR conda install
 
 v0.9.1 - 2021-09-23
+
```
Comment (znicholls) [inline CHANGELOG.rst:36]: This new line has to be deleted

**Comment by znicholls** [comment]
Seems to be formatting issues leftover for some reason

**Comment by maritsandstad** [comment]
Think all issues are solved now

File: CHANGELOG.rst [hunk 4]
```diff
@@ -14,6 +14,19 @@ The changes listed in this file are categorised as follows:
...
+----------
+
+Fixed
+~~~~~~~
+
+- (`#52 <https://github.com/openscm/openscm-runner/pull/57>` _) Updating CICERO-SCM fortran binary which confused OHC and OHC down to 700 meters in output. Also adding missing components to radiative imbalance. Updating tests to match new version.
+
+
+2021-09-17
+
```
Comment (znicholls) [inline CHANGELOG.rst:27]: @maritsandstad can you please delete this new line which shouldn't be there

File: CHANGELOG.rst [hunk 5]
```diff
@@ -14,6 +14,19 @@ The changes listed in this file are categorised as follows:
...
+2022-01-12
+----------
+
+Fixed
+~~~~~~~
+
+- (`#52 <https://github.com/openscm/openscm-runner/pull/57>` _) Updating CICERO-SCM fortran binary which confused OHC and OHC down to 700 meters in output. Also adding missing components to radiative imbalance. Updating tests to match new version.
+
+
+2021-09-17
```
Comment (znicholls) [inline CHANGELOG.rst:26]:
```suggestion
```

File: CHANGELOG.rst [hunk 6]
```diff
@@ -14,6 +14,19 @@ The changes listed in this file are categorised as follows:
     - Fixed: any bug fixes
     - Security: in case of vulnerabilities.
 
+2022-01-12
```
Comment (znicholls) [inline CHANGELOG.rst:17]:
```suggestion
master
```

**Pull request merged**